config/locales/translations/zendesk_apps_support.yml

```diff
@@ -243,6 +243,31 @@ parts:
       title: "App builder job: warning for generic secrets found in app text files. %{files} will be replaced with multiple file names"
       value: "Possible secrets found in %{files}. Consider reviewing the contents of these files before submitting your app."
       screenshot: "https://drive.google.com/file/d/179IMwzJvXD1m5u-0K499Ul3-qRBXAxnT"
+  - translation:
+      key: "txt.apps.admin.warning.app_build.insecure_http_request"
+      title: "App builder job: warning on detecting an insecure http request call in app source files"
+      value: "Possible insecure HTTP request to %{uri} in %{file}. Consider instead using the HTTPS protocol."
```

Comment:
Can we have a screenshot please? So that linguists can understand context and translate.
https://zendesk.atlassian.net/wiki/spaces/globalization/pages/401277972/Upload+screenshots+to+Google+Drive